### ISSUE TITLE
Source is not available from public SVN

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#41]
```
Please make the source available from SVN (or a git repository).
```

Original issue reported on code.google.com by `peterh...@gmail.com` on 26 Mar 2010 at 3:17

**Comment by GoogleCodeExporter** [comment]
```
I do not see any immediate benefit, at least not until more people are actively 
developing the tool?
```

Original comment by `lcam...@gmail.com` on 26 Mar 2010 at 3:37
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Isn't the source already today kept in a svn/git/hg repos somewhere? How about a
read-only copy of that somewhere publicly accessible?

In terms of benefits, then even when just following (and not contributing), 
it's an
awful lot easier to clone a repos locally and just update that instead of 
having to
manually download the latest package, unpack and so on.

I have all the projects I follow cloned locally and a small script (actually a
Rakefile but anyway) that goes through them one by one and pulls the latest 
source.
WAY easier than manually keeping track of new versions.

Please reconsider.
```

Original comment by `peterh...@gmail.com` on 26 Mar 2010 at 3:49

**Comment by GoogleCodeExporter** [comment]
```
Why not have the script download the source? Seems about as complex to invoke 
wget + 
tar as it is to invoke any other tool.

There is an overhead on my end to do this (keeping the "release", local, and 
public 
trunk in sync).
```

Original comment by `lcam...@gmail.com` on 26 Mar 2010 at 5:07

**Comment by GoogleCodeExporter** [comment]
```
This is IT, so everything is course possible. But having the full history in an 
SCM
of some kind that is designed for this kind of stuff is undeniably better than
fetching the downloads page, parsing the html, look for the download link of the
latest version, download it, untar to a new dir, diff to old.

Anyway, it's your project, you obviously do as you see fit. I just think it 
would be
easier to get it out there.

Especially since I'm assuming that this is already in an SCM system somewhere so
giving people read-only access to that (or pushing it to either google's SCM or
something like gitorious.org as part of the deployment process) so it should be 
trivial.

Up to you.
```

Original comment by `peterh...@gmail.com` on 31 Mar 2010 at 4:52

**Comment by GoogleCodeExporter** [comment]
```
I have been tracking skipfish changes in git as part of my learning process. The
results are shared on github, as there is no official repository.
http://github.com/spinkham/skipfish

New versions are usually added within a few hours of release since I track file
releases through google code.

I will happily link to the official repository instead once there is one.
```

Original comment by `steve.pi...@gmail.com` on 31 Mar 2010 at 6:57
